src/news/origins.md.draft

```diff
@@ -0,0 +1,79 @@
+---
+title: How it all started
+description: Why was PolyMC created?
+date: 2022-02-14
+tags:
+  - fork
+  - multimc
+---
+
+PolyMC is a fork of the open source Minecraft launcher [MultiMC](https://multimc.org/).
+If you don't know what a *fork* is, you can [read about it here](https://en.wikipedia.org/wiki/Fork_(software_development)).
+Many forks exist, because (part of) a community around a project is unsatisfied.
+PolyMC is also one of these forks.
+
+# Change of course with MultiMC
+The catalyst for this dissatisfaction was the account migration of Minecraft: Java Edition to Microsoft Authentication (MSA).
+By design MSA is more restrictive for third-party launchers, as it needs a private token to communicate with Microsoft servers, in contrast to the rather open nature of Mojang authentication.
+In September 2021 @peterix and the other MultiMC contributors have [released MultiMC 0.6.13][mmc-upd-2021-09].
+This version finally brought MSA support to MultiMC and thus allowed people to log into their migrated Minecraft accounts.
+Sadly this version marks the point where frustration in the community started to grow.
+
+## The introduction of `DevLauncher`
+Alongside MSA support the codebase was de-branded and all API keys were removed with version 0.6.13.
```

Comment:
it marked the start of a de-branding process

Reply:
MSA never was in 0.6.13